content/en/docs/concepts/security/index.md

```diff
@@ -630,333 +631,277 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
-in the [constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).
-In the case that the attribute is a `map`, for example `request.headers`, the
-`key` is an entry in the map, for example `request.headers[version]`.
+The example below shows a simple policy `allow-all` which allows full access to all
+workloads in the `default` namespace.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRole
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: products-viewer-version
+  name: allow-all
   namespace: default
 spec:
   rules:
-  - services: ["products.default.svc.cluster.local"]
-    methods: ["GET", "HEAD"]
-    constraints:
-    - key: request.headers[version]
-      values: ["v1", "v2"]
+  - {}
 {{< /text >}}
 
-#### `ServiceRoleBinding`
-
-A `ServiceRoleBinding` specification includes two parts:
-
--  **`roleRef`** refers to a `ServiceRole` resource in the same namespace.
--  A list of **`subjects`** that are assigned to the role.
-
-You can either explicitly specify a *subject* with a `user` or with a set of
-`properties`.  A *property* in a `ServiceRoleBinding` *subject* is similar to
-a *constraint* in a `ServiceRole` specification. A *property* also lets you use
-conditions to specify a set of accounts assigned to this role. It contains a
-`key` and its allowed *values*. The supported `key` values of a constraint
-are listed in the
-[constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).
-
-The following example shows a `ServiceRoleBinding` named
-`test-binding-products`, which binds two subjects to the `ServiceRole` named
-`"product-viewer"` and has the following `subjects`
-
-- A service account representing service **a**, `"service-account-a"`.
-- A service account representing the Ingress service
-  `"istio-ingress-service-account"` **and** where the JWT `email` claim is
-  `"a@foo.com"`.
+The example below shows a simple policy `deny-all` which denies access to all workloads
+in the `admin` namespace.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRoleBinding
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: test-binding-products
-  namespace: default
+  name: deny-all
+  namespace: admin
 spec:
-  subjects:
-  - user: "service-account-a"
-  - user: "istio-ingress-service-account"
-    properties:
-      request.auth.claims[email]: "a@foo.com"
-  roleRef:
-    kind: ServiceRole
-    name: "products-viewer"
+  {}
 {{< /text >}}
 
-In case you want to make a service publicly accessible, you can set the
-`subject` to `user: "*"`. This value assigns the `ServiceRole` to **all (both authenticated and
-unauthenticated)** users and services, for example:
+#### Custom conditions
+
+You can also use the `when` section to specify additional conditions, for example, the following
```

Comment:
You can also use the `when` section to specify additional conditions.~,~ ~f~For example, the following